src/test/modules/Runner.test.js

```diff
@@ -46,27 +46,28 @@ test('Obtain the results as an object called results with Actor name and the sta
     var actionObjects = [
         {'actor': 'Marco', 'action': 'Shoot Weapon', 'target': 'east', 'scenes': 2},
         {'actor': 'Marco', 'action': 'Shoot Weapon', 'target': 'west', 'scenes': 2},
-        {'actor': 'RAS1', 'action': 'Shoot Weapon', 'target': 'north', 'scenes': 3},
-        {'actor': 'RAT1', 'action': 'Shoot Weapon', 'target': 'south', 'scenes': 11}];
+        {'actor': 'RAS1', 'action': 'Shoot Weapon', 'target': 'north', 'scenes': 2},
+        {'actor': 'RAT1', 'action': 'Shoot Weapon', 'target': 'south', 'scenes': 2}];
 
-    expect(mainTest.follow(actorObjects, actionObjects, 5)).toEqual(
+    expect(mainTest.follow(actorObjects, actionObjects, 2)).toEqual(
         {
             'Marco': 'Alive',
             'RAS1': 'Dead',
             'RAT1': 'Dead'
         });
 });
 
-test('Obtain the results as an object called results with Actor name and the status, everyone survives', () => {
+test('Obtain the results as an object called results with Actor name and the status, everybody survives because the Runner has just 1 scene, and none action is executed', () => {
+    const mainTest1 = new Runner();
     var actorObjects = [
         {'health': 10,
             'name': 'Marco',
             'type': 'PF Squad Soldier',
             'weapon': {
                 'name': 'handgun',
-                'power': 1,
-                'xScope': 1,
-                'yScope': 1
+                'power': 2,
```

Comment:
Why is the power changed?

Reply:
prove the functionality in that specific Scenario, because in the test title says "Obtain the results as an object called results with Actor name and the status, **everybody survives because the Runner has just 1 scene, and none action is executed**"

if you see that power are enough to kill the other actors, beacuse the other actors have 2 of health, and they are just one position from the main actor (Marco) but how title says they are going to survive because the Scenario just have **1 Scene** and in the first Scene based on requirements, none action can be executed until the second scene.

tell me if it is more clear for you.

Reply:
Yeah it's clear